core/src/main/java/org/apache/gravitino/metalake/MetalakeManager.java

```diff
@@ -60,6 +67,21 @@ public class MetalakeManager implements MetalakeDispatcher {
 
   private final IdGenerator idGenerator;
 
+  @VisibleForTesting
+  static final Cache<NameIdentifier, BaseMetalake> METALAKE_CACHE =
```

Comment:
If our goal is to accelerate the acquisition of `in-use`, it seems that we only need to cache the corresponding `in-use` value, and do not need to cache BaseMetalake.

Reply:
Cacheing `BaseMetalake` will take only a little memory and can use it when `loadingMetalake` by the way.

Reply:
I think caching metalake is better, because the amount of metalake is quite limited, with small memory size we can improve the performance a lot, it is worthy to cache the metalake.

Reply:
Why do you use uppercase for this variable? Typically, we only use uppercase letter for `final` variable.